nbs/models.ipynb

```diff
@@ -97,6 +97,7 @@
    ]
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    nhits_config = {
</pre><p>You can use this comments on the to solve my last comment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/557/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='78'/>